File: app/src/main/java/com/zzl/top/common/http/IResponse.java
package com.zzl.top.common.http;

/**
 * Created by zhangzl
 * time: 2017/10/16 20:40.
 * 描述：
 */

public interface IResponse {
    /**
     * 状态码
     */
    int getCode();

    /**
     * 数据体
     */
    String getData();
}

File: http/src/main/kotlin/com/zzl/net/IResponse.kt
package com.zzl.net

/**
 * Created by zhangzl
 * 时间： 2017/10/29 14:05
 * 描述：
 */
interface IResponse {

    /**
     * 状态码
     */
    fun getCode(): Int

    /**
     * 数据体
     */
    fun getData(): String

    fun setCode(code: Int)

    fun setData(data: String?)
}